uktrade/data-hub-frontend
# Setup investment project layout

## Description of change

Now that all the investment project pages have been reactified, we no longer need the Nunjucks template. As with previous layout refactoring work, this will be done in several stages.

The following changes have been made in this PR:
- Create the `ProjectLayout` component
- Refactor `InvestmentProjectLocalHeader` to work within the new layout
- Refactor all pages that require minimal refactoring to use the new layout (including cleanup and legacy code removal). These pages have also been refactored to use React Router.

This PR is much smaller than it looks as GitHub treats indentation changes as additions. No changes have been made to any of the code for the actual pages.

## Test instructions

Go to the following pages in an investment project:

- Project details
- Project team
- Edit project details
- Edit project requirements
- Edit project value
- Project evaluation
- Project interactions
- Project propositions

They should work as before.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
